src/app/courses/course-item/course-item.component.html

```diff
@@ -1,8 +1,16 @@
-<div class="course">
+<div 
+  class="course"
+  [appFreshCourse]="course.creationDate">
   <div class='course-content'>
-    <div class="course-content-title">{{course.title}}</div>
-    <div class="course-content-duration">{{course.duration}}</div>
-    <div class="course-content-date">{{course.creationDate}}</div>
+    <div class="course-content-title">
+      {{course.title | uppercase}}
+      <fa-icon
+        [icon]="STAR_ICON"
+        [ngClass]="{'course-content-title-star': course.topRated}">
+      </fa-icon>
+    </div>
+    <div class="course-content-duration">{{course.duration | courseDuration}}</div>
+    <div class="course-content-date">{{course.creationDate | courseCreationDate}}</div>
```

Comment:
`Use built-in date pipe for course date. Change mock data to have datetime creationDate field instead of preformatted string.`

According to task date pipe should be built-in.

Reply:
changed